src/State/Pagination/Pagination.php

```diff
@@ -219,13 +219,38 @@ private function getGraphQlEnabled(?Operation $operation): bool
         return $operation?->getPaginationEnabled() ?? $enabled;
     }
 
+    /**
+     * Extract pagination parameter
+     * page[page] => $contextFilters['page'] with default configuration page_parameter_name: page
+     * page[number] => $contextFilters['_page'][number] with configuration page_parameter_name: page[number].
```

Comment:
you mean that you configure `page[number]` on the `page_parameter_name`, then you use `page[number]=1`?

Reply:
Yeah exactly

Reply:
To add more context to this, JSON:API consider query parameter such as two parameters named `page[offset]` and `page[limit]` really are two separate parameters, instead of a single page parameter in an array form.

[https://jsonapi.org/format/#query-parameters-families](https://jsonapi.org/format/#query-parameters-families)

Reply:
Yeah or `page[number]` & `page[size]` in my implementation if you work with page numbers instead of offset/limit cf https://jsonapi.org/format/#fetching-pagination

Reply:
can you add a function test and a unit test?